### ISSUE TITLE
Add WMO No. 306 Vol I.2 Common code table C-3

### ISSUE THREAD

**Issue by 6a6d74** [wmo-registers/code-manuals#6]
Common code-table C-3 _Instrument make and type for water temperature profile measurement with fall rate equation coefficients_ is required to support JCOMM/IOC activities.

Given that C-3 talks about instrument types, it is conceivable that these terms may be worth expressing as subclasses of [Sensor](http://www.w3.org/2005/Incubator/ssn/ssnx/ssn#Section_Sensor) from the Semantic Sensor Network Ontology (SSNO). However, at this point in time, we lack the understanding of the eventual requirement to _project_ terms from C-3 onto the SSNO.

In the interim, the terms from C-3 should be published as SKOS Concepts.

In expressing terms from C-3 as _both_ SKOS Concept (individual) and sub-class of ssn:Sensor (class) we "cross the streams" of A-box and T-box semantics ... which is historically problematic when implementing formal Description Logic. However, OWL2-DL supports the concept of [punning](http://www.w3.org/TR/owl2-new-features/#F12:_Punning) such that a resource can be declared as both individual and class; an OWL2-DL reasoner treats the different uses of such a resource as different entities ensuring that A-box and T-box streams don't get mixed within the reasoning.

So in summary: it's OK to define these terms as SKOS Concepts for now; we can add the complexity later.

For expedience, [BODC](http://www.bodc.ac.uk/) are planning to publish a version of C-3 via the [NERC vocabulary server](http://vocab.nerc.ac.uk/). Mappings (e.g. owl:sameAs or skos:exactMatch) from codes.wmo.int to the equivalent terms in the NERC vocabulary server should be added too.
